lib/JsModule.js

```diff
@@ -173,6 +173,7 @@ export default class JsModule {
       importFunction: config.get('importFunction', { pathToImportedModule }),
       namedImports,
       areOnlyTypes: this.isType,
+      danglingCommas: config.get('danglingCommas')
```

Comment:
This is what's causing the build failures. You need a comma at the end of the line.